queue_services/entity-emailer/src/entity_emailer/email_templates/BC-BN.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    {% if title %}
+    <title>{{ title }}</title>
+    {% else %}
+    <title>{{business.legalName}} - Business Number Information</title>
+    {% endif %}
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            <p>
+              Canada Revenue Agency has provided a Business number for the company {{ business.legalName }},
+              incorporation number {{ business.identifier }}.
+            </p>
+
+            [[whitespace-16px.html]]
+            <tr>
```

Comment:
I don't think this is going to display as you hope. This is a table-row element but you're already in a table-data block. You may need to wrap this in a table element (as in business-info.html), or just a simple paragraph + colon + BN. The requirements doc leaves this open to your design.

Reply:
thanks for the tip!

Reply:
I meant fixing the `<tr>` block, not deleting the whitespace part.

Reply:
haha okay that makes sense

Reply:
okay I think this is actually what I was going for

Reply:
That should work :)